javascripts/discourse/templates/components/d-navigation.hbs

```diff
@@ -0,0 +1,52 @@
+{{bread-crumbs
```

Comment:
I think we have to rename this component, but just a suggestion. Eventually, we might come up with an actual breadcrumbs component since this is more of a category bar

Reply:
this is a discourse built-in I miss the link to the d-navigation I will add it.